models/lfs_lock.go

```diff
@@ -0,0 +1,141 @@
+// Copyright 2017 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package models
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	api "code.gitea.io/sdk/gitea"
+)
+
+// LFSLock represents a git lfs lock of repository.
+type LFSLock struct {
+	ID      int64     `xorm:"pk autoincr"`
+	RepoID  int64     `xorm:"INDEX"`
+	Owner   *User     `xorm:"-"`
+	OwnerID int64     `xorm:"INDEX"`
+	Path    string    `xorm:"TEXT"`
```

Comment:
And maybe `Path` should be unique with `RepoID` ?

Reply:
Yes and that make me think that I should also use filepath.Clean before adding a entry.

Reply:
Done + add not null constraint

Reply:
Revert back the uniqueness since mysql need a fixed size to index uniquess on field and path is a TEXT field. Uniqueness is preserve by code.

Reply:
`Path` could be `xorm:"varchar(1024)"`.

Reply:
after looking at it linux limits path length to 4096